admin/schema.php

```diff
@@ -770,4 +770,12 @@ function installer_db_now() {
 
 # Release marker: 1.3.0-rc.1
 
+$g_upgrade[202] = array( 'AlterColumnSQL', array( db_get_table( 'api_token' ), "
+	user_id					I		UNSIGNED NOTNULL DEFAULT '0',
+	name					C(128)	NOTNULL DEFAULT \" '' \",
+	hash					C(128)	NOTNULL DEFAULT \" '' \",
+	date_created			I		UNSIGNED NOTNULL DEFAULT '1',
```

Comment:
Do we have a reason for defaulting dates to 1 instead of 0 other than consistency?  This seems like a strange default value.  You will need two things to make this change correct:
- API Token has a check for field not set for the case where tokens aren't used yet.  Make sure this check considers <= 1 to be never used.
- You may want to have upgrade steps to normalize already existing instances (if we care enough to do so). i.e. replace values where date == 0 with date = 1.